backend/src/packages/users/user.repository.ts

```diff
@@ -31,8 +42,17 @@ class UserRepository implements Repository {
       })
       .returning('*')
       .execute();
+    const user_details = await this.userDetailsModel
+      .query()
+      .insert({
+        fullName,
+      })
+      .returning('*')
+      .execute();
+
+    await user_details.$relatedQuery('user').relate(user);
```

Comment:
Use [upsertGraph](https://vincit.github.io/objection.js/api/query-builder/mutate-methods.html#upsertgraph)

Reply:
Sorry, I mean insertGraph, miss this one)